cdk/apps/ica_credentials/app.py

```diff
@@ -47,7 +48,25 @@
     ICA_BASE_URL,
     SLACK_HOST_SSM_NAME,
     SLACK_WEBHOOK_SSM_NAME,
-    env=cdk.Environment(account="472057503814", region="ap-southeast-2"),
+    env=cdk.Environment(
+        account="472057503814",
+        region="ap-southeast-2"
+    ),
+)
+
+# V2 (single token)
+IcaCredentialsDeployment(
+    app,
+    f"{CDK_APP_NAME}-dev-v2",
+    None,  # Token does not require project context in v2
+    None,  # Token does not require additional project list in v2
+    ICAV2_BASE_URL,
+    SLACK_HOST_SSM_NAME,
+    SLACK_WEBHOOK_SSM_NAME,
+    env=cdk.Environment(
```

Comment:
So is there going to be no difference between our ica v2 settings between *our* prod and dev?

Reply:
I don't know. For now probably not, and we don't have a prod account? I left a comment regarding the `encoded_params` since we can specify a tenant in V2 but we currently only have the one.  There may be potential to restrict the scope of the api key? So we could have one api key that is part of workgroup 'dev-admin' which has admin access to the dev project but only read access to the prod project? 

@reisingerf @victorskl thoughts?

Reply:
The proper ICA dev/prod isolation should be at tenant level, TBH. Different projects within the same ICA tenant still share the compute environment. At project level, we only can achieve "data" isolation.

Likewise in v1 and GDS; data will be still scoped into Project in v2 i.e. `icav2 projectdata list`. It is just that, project scoped for data is explicit in v1 (i.e. we assign who can write/read to GDS volume) but implicit in v2 (creating a project implicitly get data volume). So, there's still a dedicated project needed for the production pipeline, I suppose.